plugin/vcs/gitlab/gitlab.go

```diff
@@ -182,11 +182,16 @@ func (provider *Provider) APIURL(instanceURL string) string {
 	return fmt.Sprintf("%s/%s", instanceURL, apiPath)
 }
 
-// TryLogin will try to login GitLab.
```

Comment:
I might prefer to have two different methods for different purposes, TryLogin + FetchUserInfo two methods.